Source/WebInspectorUI/UserInterface/Views/InlineSwatch.js

```diff
@@ -362,28 +400,24 @@ WI.InlineSwatch = class InlineSwatch extends WI.Object
         this.dispatchEventToListeners(WI.InlineSwatch.Event.Activated);
 
         switch (this._type) {
-        case WI.InlineSwatch.Type.CubicBezierTimingFunction:
-            this._valueEditor.cubicBezierTimingFunction = value;
-            break;
-
-        case WI.InlineSwatch.Type.BoxShadow:
-            this._valueEditor.boxShadow = value;
-            break;
-
         case WI.InlineSwatch.Type.Color:
-            this._valueEditor.color = value;
             this._valueEditor.focus();
             break;
 
         case WI.InlineSwatch.Type.Gradient:
             this._valueEditor.gradient = value;
             break;
 
-        case WI.InlineSwatch.Type.SpringTimingFunction:
-            this._valueEditor.springTimingFunction = value;
-            break;
 
         case WI.InlineSwatch.Type.Variable: {
+            if (this._variableType) {
+                let colorVariable = this._delegate?.inlineSwatchColorVariableCompute(this, value.toString());
```

Comment:
ditto above about `this._delegate?.inlineSwatchGetMatchingColorVariable?.(this, value.toString)) || null`